### PR TITLE
LPS-117358 Change how "subypes" are obtained for Display Page Templates to leverage FormVariations Insert Lucidchart Diagram Export

### PR DESCRIPTION
**Pull request by jkappler**

### CODE REVIEW

**Comment by jkappler** [comment]
ci:test:relevant

**Comment by jkappler** [comment]
ci:test:sf

**Comment by jkappler** [comment]
@JorgeFerrer

File: modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/display/context/ContentPageEditorLayoutPageTemplateDisplayContext.java [hunk 1]
```diff
@@ -177,15 +177,19 @@ private String _getMappingTypeLabel() {
...
 
-		if (infoDisplayContributor == null) {
+		if (infoItemFormProvider == null) {
 			return null;
 		}
 
-		return infoDisplayContributor.getLabel(themeDisplay.getLocale());
+		InfoForm infoForm = infoItemFormProvider.getInfoForm();
+
+		return ResourceActionsUtil.getModelResource(
```
Comment (JorgeFerrer) [inline modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/display/context/ContentPageEditorLayoutPageTemplateDisplayContext.java:191]: @jkappler it's better to use the getInfoItemClassDetails() method to obtain a bean representing the classes supported by the service (InfoItemFormProvider in this case) including the getLabel() method that you need.

**Comment by jkappler** [comment]
ci:forward

File: modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/display/context/ContentPageEditorLayoutPageTemplateDisplayContext.java (hunk 1 above)
Comment (jkappler) [reply modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/display/context/ContentPageEditorLayoutPageTemplateDisplayContext.java:191]: will do so

Comment (jkappler) [reply modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/display/context/ContentPageEditorLayoutPageTemplateDisplayContext.java:191]: https://github.com/liferay-echo/liferay-portal/pull/1630

**Pull request closed without merging**